middleware/locals.js

```diff
@@ -4,120 +4,120 @@ const moment = require('moment');
 const { map, omitBy } = require('lodash');
 
 const { getCurrentUrl, getAbsoluteUrl, localify } = require('../modules/urls');
+const { REBRAND_SECRET } = require('../modules/secrets');
+const appData = require('../modules/appData');
 const routes = require('../controllers/routes');
 
-/**
- * Get normalised page title for metadata
- */
-function getMetaTitle(base, pageTitle) {
-    return pageTitle ? `${pageTitle} | ${base}` : base;
-}
+const cookies = config.get('cookies');
+const features = config.get('features');
 
 /**
  * Set request locals
  * - Local properties that depend on the request
  * - Local methods for use in views that depend on the request
  */
-module.exports = {
-    // Export for tests
-    getMetaTitle,
-    // Export middleware
-    middleware: function(req, res, next) {
-        const locale = req.i18n.getLocale();
-
-        /**
-         * Navigation sections for top-level nav
-         */
-        const itemsToShow = omitBy(routes.sections, s => s.showInNavigation === false);
-        res.locals.navigationSections = map(itemsToShow, (section, id) => {
-            return {
-                id: id,
-                path: localify(locale)(section.path),
-                label: req.i18n.__(section.langTitlePath)
-            };
-        });
-
-        /**
-         * High-contrast mode
-         */
-        const contrastPref = req.cookies[config.get('cookies.contrast')];
-        res.locals.isHighContrast = contrastPref && contrastPref === 'high';
-
-        /**
-         * Features
-         */
-        res.locals.enablePrompt = config.get('features.enablePrompt');
-        res.locals.enableSurvey = config.get('features.enableSurvey');
-
-        /**
-         * Metadata (e.g. global title, description)
-         */
-        res.locals.metadata = {
-            title: req.i18n.__('global.brand.title'),
-            description: req.i18n.__('global.brand.description')
+module.exports = function(req, res, next) {
+    const locale = req.i18n.getLocale();
+
+    /**
+     * Feature flags
+     */
+    res.locals.enablePrompt = features.enablePrompt;
+    res.locals.enableSurvey = features.enableSurvey;
+
+    /**
+     * High-contrast mode
+     */
+    const contrastPref = req.cookies[cookies.contrast];
+    res.locals.isHighContrast = contrastPref && contrastPref === 'high';
+
+    /**
+     * Rebrand flag
+     */
+    const useNewBrand = appData.isNotProduction && req.cookies[cookies.rebrand] === REBRAND_SECRET;
+    res.locals.useNewBrand = useNewBrand;
+
+    /**
+     * Global copy
+     */
+    const globalCopy = {
```

Comment:
Replaced all references to `{{ __('global.brand.*') }}` in the templates with this local, allows us to wholesale swap out the copy, will look at doing this for `global.contact` too once the update branch has been merged in (as it's been changed there).